### DISCUSSION TITLE
I would like to know how code-server block chrome hot keys

### DISCUSSION THREAD

**Discussion by patialo** [coder/code-server#5497] [Q&A] [answered]
I'm interesting in how code-server block chrome hot keys，I want my web page to have this function, but I haven't found a way。
can someone tell me about this？thanks a million.

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
I think it's mostly a TypeScript module absorbing keypresses from the system, so code-server can use all the keys for shortcuts. I don't know how though, but you can try searching Google, now that you might have a general idea.

**Reply by patialo** [reply]
> I think it's mostly a TypeScript module absorbing keypresses from the system, so code-server can use all the keys for shortcuts. I don't know how though, but you can try searching Google, now that you might have a general idea.

 I'm extremely grateful to you，maybe input capture library like hotkeys.js can solve my problem
